examples/fairmotion_interface.py

```diff
@@ -0,0 +1,478 @@
+        translation: Optional[mn.Vector3] = None,
+    ) -> None:
+        """
+        A method that will take attributes of the model data sets as arguments, filling in
+        empty args with the default set of data, and adds the data to the meta_data library.
+        """
+
+        data = None
+        # if name is default, set data for "default"
+        # else if name is in metadata, set data values to args or default values
+        if name in self.metadata:
+            data = self.metadata[name]
+
+        # else if name not in metadata, build data from name, args, default values, and add to lib
+        else:
+            self.metadata[name] = {}
+            data = self.metadata[name]
+
+        data["urdf_path"] = urdf_path or self.metadata["default"]["urdf_path"]
+        data["amass_path"] = amass_path or self.metadata["default"]["amass_path"]
+        data["bm_path"] = bm_path or self.metadata["default"]["bm_path"]
+        data["rotation"] = rotation or self.metadata["default"]["rotation"]
+        data["translation"] = translation or self.metadata["default"]["translation"]
+
+    def metadata_parser(self, metadata_dict: Dict[str, Any], to_file: bool):
+        """
+        Convert metadata dict to and from a form that is json serializable
+        """
+        data = metadata_dict.copy()
+
+        if to_file:
+            for k, v in data.items():
+                if isinstance(v, mn.Quaternion):
+                    q_list = [list(v.vector)]
+                    q_list.append(v.scalar)
+                    data[k] = q_list
+                elif isinstance(v, mn.Vector3):
+                    data[k] = list(v)
+
+        elif not to_file:
+            for k, v in data.items():
+                if k == "rotation":
+                    data[k] = mn.Quaternion(v)
+                elif k == "translation":
+                    data[k] = mn.Vector3(v)
+        return data
+
+    def save_metadata(self, name: str):
+        """
+        Saves the current metadata to a json file in given file path
+        """
+        filepath = f"{self.metadata_dir}{name + FILE_SUFFIX}.json"
+        data = {name: self.metadata_parser(self.metadata[name], to_file=True)}
+
+        with open(filepath, "w") as file:
+            json.dump(data, file)
+
+    def fetch_metadata(self, name):
+        """
+        Fetch metadata from a json file in given file name and sets current metadata
+        """
+        if FILE_SUFFIX in name and len(name) > len(FILE_SUFFIX):
+            # if name give is full name of file, truncate FILE_SUFFIX
+            filename = name
+            name = name[: -len(FILE_SUFFIX)]
+        else:
+            # including FILE_SUFFIX if not included in name
+            filename = name + FILE_SUFFIX
+
+        filepath = f"{self.metadata_dir}{filename}.json"
+
+        try:
+            with open(filepath, "r") as file:
+                data = json.load(file)
+        except Exception:
```

Comment:
These exception types are too broad? Handling a base exception like this is really bad. Did you intend to catch an `IOError`?